image/Dockerfile

```diff
@@ -540,8 +540,8 @@ RUN python3 -m py_compile /usr/libexec/pyrex/*.py
 
 # Remove all non-root users and groups so that there are no conflicts when the
 # user is added
-RUN getent passwd | cut -f1 -d: | grep -v '^root$' | xargs -L 1 userdel
-RUN getent group | cut -f1 -d: | grep -v '^root$' | xargs -L 1 groupdel
+RUN for user in `getent passwd | cut -f1 -d: | grep -v '^root$'`; do userdel $user; done
+RUN for group in `getent group | cut -f1 -d: | grep -v '^root$'`; do groupdel $group; done
```

Comment:
Would it be better to do: `xargs -r -L 1 ...` ?

Reply:
Sure